hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/googlepubsub/MessageCompressor.java

```diff
@@ -17,15 +17,18 @@ byte[] compress(byte[] data) throws IOException {
         Codec codec = codecFactory.createInstance();
 
         ByteBuffer compressed = codec.compress(ByteBuffer.wrap(data));
+        byte[] compressedBytes = new byte[compressed.limit()];
```

Comment:
This can potentially cause some CPU increase, look at that if there will be a problem